### ISSUE TITLE
The error is significant

### ISSUE THREAD

**Issue by yuanfanglila** [AdityaLab/pinnsformer#8]
May I ask why there is a significant error in the reproduction results, which did not achieve the accuracy shown in the paper

**Comment by tsano430** [comment]
FYI: https://github.com/AdityaLab/pinnsformer/issues/6#issuecomment-2407857816

**Comment by sea-dragon1** [comment]
me too. I trained according to the instructions, but the loss function could not go down, it was about 0.2, but your result loss can be reduced to 0.00001.

**Comment by tsano430** [comment]
Example: https://github.com/AdityaLab/pinnsformer/issues/5#issuecomment-2635459293

**Comment by My-git96** [comment]
Obviously, it's a fake work
